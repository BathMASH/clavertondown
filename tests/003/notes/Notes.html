<!DOCTYPE html>
<html lang="en" xml:lang="en">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Test 003: Repeat an environment with the correct numbering" />
<meta property="og:type" content="book" />





<meta name="author" content="Emma Cliffe, Skills Centre: MASH, University of Bath" />



<meta name="description" content="Test 003: Repeat an environment with the correct numbering">

<title>Test 003: Repeat an environment with the correct numbering</title>

<script src="libs/header-attrs-2.3/header-attrs.js"></script>
<script src="libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="libs/navigation-1.1/tabsets.js"></script>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
</style>

<style type="text/css">
  a {
   color: blue;
  }
</style>

</head>

<body>

<div class="container-fluid main-container">


<!--bookdown:title:start-->
<div id="header">
<h1 class="title">Test 003: Repeat an environment with the correct numbering</h1>
<h4 class="author"><em>Emma Cliffe, Skills Centre: MASH, University of Bath</em></h4>
<h4 class="date"><em>August 2020</em></h4>
</div>
<!--bookdown:title:end-->

<!--bookdown:toc:start-->
<div id="TOC">
<ul>
<li><a href="#section-1"><span class="toc-section-number">1</span> Section 1</a></li>
<li><a href="#section-2"><span class="toc-section-number">2</span> Section 2</a></li>
</ul>
</div>
<!--bookdown:toc:end-->

<!--bookdown:body:start-->
<div id="section-1" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Section 1</h1>

<div id="exm:inbuilt" class="bookdown-example" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;">
<span id="exm:inbuilt" class="example" custom-style="NameStyle"><strong>Example 1.1:  </strong></span>
<p>
2+3
</p>
</div>

<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle">
<span id="Exe:author" class="Exercise" custom-style="NameStyle"><strong> Exercise 1.1:  </strong></span>
<p>
4+5
</p>
</div>

<div id="exm:inbuilt" class="bookdown-example" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;">
<span id="exm:inbuilt" class="example" custom-style="NameStyle"><strong>Example 1.1:  </strong></span>
<p>
2+3
</p>
</div>
</div>
<div id="section-2" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Section 2</h1>

<div id="exm:unnamed-chunk-1" class="bookdown-example" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;">
<span id="exm:unnamed-chunk-1" class="example" custom-style="NameStyle"><strong>Example 2.1:  </strong></span>
<p>
I should be example 2.1
</p>
</div>

<div id="exm:inbuilt" class="bookdown-example" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;">
<span id="exm:inbuilt" class="example" custom-style="NameStyle"><strong>Example 1.1:  </strong></span>
<p>
2+3
</p>
</div>

<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle">
<span id="Exe:author" class="Exercise" custom-style="NameStyle"><strong> Exercise 1.1:  </strong></span>
<p>
4+5
</p>
</div>

<div id="exm:unnamed-chunk-2" class="bookdown-example" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;">
<span id="exm:unnamed-chunk-2" class="example" custom-style="NameStyle"><strong>Example 2.2:  </strong></span>
<p>
I should be example 2.2
</p>
</div>

<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle">
<span id="Exe:author2" class="Exercise" custom-style="NameStyle"><strong> Exercise 2.1:  </strong></span>
<p>
I should be exercise 2.1
</p>
</div>
<p>Conclusion: This worked already everywhere except LaTeX. This is a Bookdown issue. To be fair, there isnâ€™t any nice way to do this in LaTeX. I did not want to go down the road of restatable in LaTeX. Instead I wanted to resolve this prior to that point in the way an author would. To do so we change the repeated label to be the ref instead and also ensure that the counter is not incremented.</p>
<!--chapter:end:index.Rmd-->
</div>
<!--bookdown:body:end-->


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
