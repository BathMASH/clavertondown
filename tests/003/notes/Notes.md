---
title: "Test 003: Repeat an environment with the correct numbering"
author: 'Emma Cliffe, Skills Centre: MASH, University of Bath'
date: 'August 2020'
site: bookdown::bookdown_site
lang: en
documentclass: article
classoption: a4paper
fontsize: 10pt
geometry: margin=2.5cm
output:
  clavertondown::word_clav:
    toc: true
    number_sections: true
    keep_md: true
  clavertondown::epub_clav:
    toc: true
  clavertondown::gitbook_clav:
    split_by: section
    self_contained: no
    config:
      download: [["Notes.html", "HTML page"], ["Notes.pdf","Standard print PDF"], ["NotesClear.pdf","Clear print PDF"], ["NotesLarge.pdf","Large print PDF"], ["Notes.docx","Accessible Word document"], ["Notes.epub","Accessible EPub book" ]]
      sharing: no
  clavertondown::pdf_clav:
    latex_engine: pdflatex
    dev: pdf
    keep_tex: true
    keep_md: true
    fig_caption: true
    toc: true
  clavertondown::html_clav:
    toc: true
    fig_caption: true
    keep_md: true
---

# Section 1





\BeginKnitrBlock{example}<div class="bookdown-example" id="exm:inbuilt" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;"><span class="example" id="exm:inbuilt" custom-style="NameStyle"><strong>(\#exm:inbuilt) </strong></span><p>2+3</p></div>\EndKnitrBlock{example}

\BeginKnitrBlock{Exercise}<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle"><span class="Exercise" id="Exercise:author" custom-style="NameStyle"><strong> Exercise (\#Exercise:author) </strong></span><p>4+5</p></div>\EndKnitrBlock{Exercise}

\BeginKnitrBlock{example}<div class="bookdown-example" id="exm:inbuilt" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;"><span class="example" id="exm:inbuilt" custom-style="NameStyle"><strong>(\#exm:inbuilt) </strong></span><p>2+3</p></div>\EndKnitrBlock{example}

# Section 2

\BeginKnitrBlock{example}<div class="bookdown-example" id="exm:unnamed-chunk-1" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;"><span class="example" id="exm:unnamed-chunk-1" custom-style="NameStyle"><strong>(\#exm:unnamed-chunk-1) </strong></span><p>I should be example 2.1</p></div>\EndKnitrBlock{example}

\BeginKnitrBlock{example}<div class="bookdown-example" id="exm:inbuilt" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;"><span class="example" id="exm:inbuilt" custom-style="NameStyle"><strong>(\#exm:inbuilt) </strong></span><p>2+3</p></div>\EndKnitrBlock{example}

\BeginKnitrBlock{Exercise}<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle"><span class="Exercise" id="Exercise:author" custom-style="NameStyle"><strong> Exercise (\#Exercise:author) </strong></span><p>4+5</p></div>\EndKnitrBlock{Exercise}

\BeginKnitrBlock{example}<div class="bookdown-example" id="exm:unnamed-chunk-2" custom-style="ExampleStyle" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;"><span class="example" id="exm:unnamed-chunk-2" custom-style="NameStyle"><strong>(\#exm:unnamed-chunk-2) </strong></span><p>I should be example 2.2</p></div>\EndKnitrBlock{example}


\BeginKnitrBlock{Exercise}<div class="Exercise" style="margin-bottom: 1.5em; margin-top:1.5em; background-color: lavenderblush; border-left-style: solid; border-color: mediumorchid; padding-left: 0.5em;" custom-style="ExampleStyle"><span class="Exercise" id="Exercise:author2" custom-style="NameStyle"><strong> Exercise (\#Exercise:author2) </strong></span><p>I should be exercise 2.1</p></div>\EndKnitrBlock{Exercise}

Conclusion: This worked already everywhere except LaTeX. This is a Bookdown issue. To be fair, there isn't any nice way to do this in LaTeX. I did not want to go down the road of restatable in LaTeX. Instead I wanted to resolve this prior to that point in the way an author would. To do so we change the repeated label to be the ref instead and also ensure that the counter is not incremented. 

<!--chapter:end:index.Rmd-->

