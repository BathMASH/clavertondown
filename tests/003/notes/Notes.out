\BOOKMARK [1][-]{section.1}{\376\377\000S\000e\000c\000t\000i\000o\000n\000\040\0001}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000S\000e\000c\000t\000i\000o\000n\000\040\0002}{}% 2
